modules/imgproc/src/stackblur.cpp

```diff
@@ -0,0 +1,845 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+// Copyright (C) 2019-2021, Shenzhen Institute of Artificial Intelligence and
+// Robotics for Society, all rights reserved.
+// Third party copyrights are property of their respective owners.
+
+#include "precomp.hpp"
+#include "opencv2/core/hal/intrin.hpp"
+
+static unsigned short const stackblur_mul[255] =
```

Comment:
I guess, you based your implementation on some other code. What's the license of the original implementation? At least, you should put the link/reference to the original code, copyright and information about the license (or the license itself, if it's short)

Reply:
Hi, this implementation is based on two projects ([project1](https://github.com/flozz/StackBlur) and [project2](http://vitiy.info/Code/stackblur.cpp)).  Should I Change the Copy Right like the [following](https://github.com/flozz/StackBlur/blob/master/LICENSE-MIT.txt)?:
``
Copyright (c) 2010 Mario Klingemann
``